frontend/src/helpers/index.ts

```diff
@@ -306,9 +306,12 @@ function getProductName(): string | undefined {
   return process.env.REACT_APP_HEADLAMP_PRODUCT_NAME;
 }
 
+export const DEFAULT_DROP_SHELL_LINUX_IMAGE = 'docker.io/alpine:3.19';
```

Comment:
Should we use the `latest` tag? I guess using the latest tag we risk users having to pull when the image is not around but has the advantage that we don't have to keep updating it?

Reply:
Addressed. Resolving.